src/nunjucks/template.njk

```diff
@@ -22,11 +22,13 @@ Component options:
     }] 
   - activeLinkId (string): The id of the link that is currently active 
   - oneLoginLink (string): Option to overwrite the default One Login Home link. Defaults to "https://home.account.gov.uk/"
-  - homepageLink (string): Option to overwrite the default GOV.UK link. Defaults to "https://www.gov.uk/"
+  - signOutLink (string): Option to overwrite the default signed out page. Defaults to "https://home.account.gov.uk/"
```

Comment:
This should be `Defaults to "https://home.account.gov.uk/sign-out"` while `homepageLink` should be `Defaults to "https://home.account.gov.uk/"`

Reply:
Opps. Fixed, thanks!